src/base_station/debug_console.cpp

```diff
@@ -34,10 +34,15 @@ struct DebugLine
     float r = 0.0f, g = 1.0f, b = 0.0f;
 };
 
-struct Waypoints
+struct Waypoint
 {
-	//Each element is a vector with two elements: [Latitude, Longitude]
-	std::vector<std::vector<double>> wpList;
+	float Latitude;
+	float Longitude;
+};
+
+struct wayPointList
```

Comment:
Instead of doing this wayPointList thing, just do
```
std::vector<Waypoint> wpList;
```
Then you'll have a global variable named `wpList`.